components/constellation/constellation.rs

```diff
@@ -397,8 +396,8 @@ pub struct Constellation<STF, SWF> {
     /// and the namespaces are allocated by the constellation.
     next_pipeline_namespace_id: PipelineNamespaceId,
 
-    /// Bits of state used to interact with the webdriver implementation
-    webdriver: WebDriverData,
+    // Forward responses from the script thread to the webdriver server.
```

Comment:
Oh, sorry, just noticed that this is no longer rustdoc. Removing the extra slash here means this documentation doesn't  show up on hover in the IDE.

```suggestion
    /// An [`IpcSender`] which forwards responses from the `ScriptThread` to the WebDriver server.
```